!article/article.md

```diff
@@ -0,0 +1,179 @@
+# Introduction
+[OpenSSF Scorecard](https://github.com/ossf/scorecard) is a tool developed by the Open Source Security Foundation (OpenSSF) that provides automated security assessments for open-source projects. The primary goal of the Scorecard project is to help developers and users determine the security posture of open-source software by generating a **score** based on a series of security-related checks. 
+
+![nuget](./Scorecard-NuGet.png)
```

Comment:
@Ioana37 this image is rendering very big, would it possible to have it smaller?